datasets/LCCusePriceEscalationDataSet2019.idf

```diff
@@ -1,2606 +1,2539 @@
-    1.0394,                              !- Year 2 Escalation
-    1.0394,                              !- Year 3 Escalation
-    1.0424,                              !- Year 4 Escalation
-    1.0394,                              !- Year 5 Escalation
-    1.0424,                              !- Year 6 Escalation
-    1.0485,                              !- Year 7 Escalation
-    1.0545,                              !- Year 8 Escalation
-    1.0545,                              !- Year 9 Escalation
-    1.0576,                              !- Year 10 Escalation
-    1.0636,                              !- Year 11 Escalation
-    1.0667,                              !- Year 12 Escalation
-    1.0697,                              !- Year 13 Escalation
-    1.0697,                              !- Year 14 Escalation
-    1.0697,                              !- Year 15 Escalation
-    1.0636,                              !- Year 16 Escalation
-    1.0667,                              !- Year 17 Escalation
-    1.0667,                              !- Year 18 Escalation
-    1.0697,                              !- Year 19 Escalation
-    1.0727,                              !- Year 20 Escalation
-    1.0758,                              !- Year 21 Escalation
-    1.0758,                              !- Year 22 Escalation
-    1.0788,                              !- Year 23 Escalation
-    1.0788,                              !- Year 24 Escalation
-    1.0818,                              !- Year 25 Escalation
-    1.0848,                              !- Year 26 Escalation
-    1.0879,                              !- Year 27 Escalation
-    1.0879,                              !- Year 28 Escalation
-    1.0909,                              !- Year 29 Escalation
-    1.0939;                              !- Year 30 Escalation
\ No newline at end of file
+! The source of the values for the following objects is:
+!
+!    NISTIR 85-3273-34
+!
+!    ENERGY PRICE INDICES AND DISCOUNT FACTORS
+!    FOR LIFE-CYCLE COST ANALYSIS - 2019
+!
+!    Annual Supplement to NIST Handbook 135
+!
+!    Priya D. Lavappa
+!    Joshua D. Kneifel
+!
+! These are factors for the United States and regions of the United States.
+!
+! The factors are from a file available upon request from NIST called ENCOSTxx.TXT and are
+! converted into EnergyPlus objects using scripts/dev/update_nist_escalation.py
+! https://www.energy.gov/eere/femp/building-life-cycle-cost-programs
+! https://www.energy.gov/sites/prod/files/2019/05/f62/encost19.txt
+
+LifeCycleCost:UsePriceEscalation,
+    NorthEast Residential-Electricity,  !- Name
+    Electricity,             !- Resource
+    2020,                    !- Escalation Start Year
+    January,                 !- Escalation Start Month
+    1.0138,                  !- Year 1  Escalation [2020]
+    1.0239,                  !- Year 2  Escalation [2021]
+    1.0336,                  !- Year 3  Escalation [2022]
```

Comment:
I redid the 2019 one with the script, to ensure it wasn't producing any actual diffs (it isn't)

Reply:
the format is slightly different (and field names are more explicit) but that's it.